docs/source/usage/config.rst

```diff
@@ -5,8 +5,13 @@ Configuration
 
 .. note::
 
-   SAGA has been designed as a zero-configuration library. Unless you are 
-   experiencing problems with one of the default configuration settings, there's 
+   This section is outdated!
+
```

Comment:
We should open a ticket with medium to severe priority about updating it. SAGA is in maintenance mode so I think that outdated documentation is a greater problem than with a rapidly changing code base.

Reply:
Ack, see #760